src/app/static/src/js/components/runWorkflow/workflowSummary/runWorkflowSummary.vue

```diff
@@ -2,8 +2,7 @@
     <div id="summary-header">
         <run-workflow-summary-header :workflow-summary="workflowSummary"></run-workflow-summary-header>
         <div v-if="hasDependenciesLength">
-            <workflow-summary-reports :workflow-summary="workflowSummary"
-                                      :git-commit="workflowMetadata.git_commit"/>
```

Comment:
This was throwing an error whenever `git_commit` null. But looks like this component doesn't actually need/use this at all, so have just removed it.